### PR TITLE
Show action data after action creation

### PR DESCRIPTION
**Pull request by TomaszChmielPagepro**
Show action data in sidebar after action creation

### CODE REVIEW

File: src/components/frame/LoadingTemplate/LoadingTemplate.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ const MSG = defineMessages({
 });
 
 const delayedLoadingDuration = 15 * 1000; // 15 seconds
-export const failedLoadingDuration = 30 * 1000; // 30 seconds
+export const failedLoadingDuration = 3000 * 1000; // 30 seconds
```
Comment (KornelKwiatkowski) [inline src/components/frame/LoadingTemplate/LoadingTemplate.tsx:38]: 3000 * 1000 isn't 30 seconds.

File: src/components/v5/common/ActionSidebar/hooks/useUserHasPermissions.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import { getAllUserRoles } from '~transformers';
+
+export const useUserHasPermissions = (): boolean => {
+  const { isVotingReputationEnabled } = useEnabledExtensions();
+  const { colony } = useColonyContext();
+  const { wallet } = useAppContext();
+
+  const requiredRoles: ColonyRole[] = [ColonyRole.Root];
+
+  const hasReputation = useColonyHasReputation(colony?.colonyAddress as string);
```
Comment (KornelKwiatkowski) [inline src/components/v5/common/ActionSidebar/hooks/useUserHasPermissions.ts:20]: why do we use type assertion here?

File: src/components/v5/common/ActionSidebar/hooks/useUserHasPermissions.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  const allUserRoles = useTransformer(getAllUserRoles, [
+    colony,
+    wallet?.address,
+  ]);
+
+  const [hasRoles] = useDialogActionPermissions(
+    colony as Colony,
```
Comment (KornelKwiatkowski) [inline src/components/v5/common/ActionSidebar/hooks/useUserHasPermissions.ts:28]: same as above

File: src/components/frame/LoadingTemplate/LoadingTemplate.tsx (hunk 1 above)
Comment (TomaszChmielPagepro) [reply src/components/frame/LoadingTemplate/LoadingTemplate.tsx:38]: This was for testing, I will revert that

**Review by KornelKwiatkowski — APPROVED** [review]

**Review by joanna-pagepro — APPROVED** [review]

**Pull request merged**